### PR TITLE
feat: api v2, allow multiple input sirets

### PR DESCRIPTION
**Pull request by revolunet**
allow to pass multiple sirets in input so we can prevent multiple calls

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -20,6 +22,17 @@ app.get("/api/v1/:siret", (req, res) => {
   return res.json(results);
 });
 
+// 82161143100015,82161143100016,82161143100017
+app.get("/api/v2/:sirets", (req, res) => {
+  const sirets = req.params.sirets;
+  const nums = sirets.split(",");
+  const results = nums.map(num => ({
```
Comment (lionelB) [inline src/index.js]: Should we add a `.filter(Boolean)` before `.map` to prevent empty results when having "siret1,siret2," ?

Comment (revolunet) [reply src/index.js]: its done a little bit later so you get as much results as you asked, even if empty

Comment (revolunet) [reply src/index.js]: makes sense ?

Comment (lionelB) [reply src/index.js]: yep thanks !

**Pull request merged**